src/components/loading/loading_spinner.tsx

```diff
@@ -16,19 +16,31 @@ import { euiLoadingSpinnerStyles } from './loading_spinner.styles';
 export const SIZES = ['s', 'm', 'l', 'xl', 'xxl'] as const;
 export type EuiLoadingSpinnerSize = typeof SIZES[number];
 
+export type EuiLoadingSpinnerColor = {
+  border?: string;
+  highlight?: string;
```

Comment:
```suggestion
  border?: CSSProperties['border-color'];
  highlight?: CSSProperties['border-color'];
```

I know it evaluates to the same thing (string), but it makes it clearer the intention of where the color will be applied.